packages/waku/src/lib/plugins/vite-plugin-rsc-transform.ts

```diff
@@ -670,7 +675,10 @@ export function rscTransformPlugin(
     }
     throw new Error('server id not found: ' + id);
   };
-  const wakuDist = joinPath(fileURLToFilePath(import.meta.url), '../../..');
+  const wakuDist = joinPath(
```

Comment:
We have `wakuDist` in packages/waku/src/lib/middleware/dev-server-impl.ts too. Should it follow too?

Reply:
Tried, but not working, I think all rest path code rely on the starting `/`

Reply:
We might need fix `packages/waku/src/lib/middleware/dev-server-impl.ts` in another PR? it might takes longer time

Reply:
Interesting. It's probably something different. No hurry, but if we could understand the difference better, it might be helpful for the future.